Makefile

```diff
@@ -167,11 +167,14 @@ $(TEST_DIR)/issue_70/optimized-grammar/issue_70.go: $(TEST_DIR)/issue_70/issue_7
 	$(BINDIR)/pigeon -nolint -optimize-grammar $< > $@
 
 $(TEST_DIR)/issue_70b/issue_70b.go: $(TEST_DIR)/issue_70b/issue_70b.peg $(BINDIR)/pigeon
-	$(BINDIR)/pigeon -nolint --optimize-grammar $< > $@
+	$(BINDIR)/pigeon -nolint -optimize-grammar -support-left-recursion $< > $@
```

Comment:
Because `D <- B / D` is left recursion

Reply:
Ah yes. The problem did not become apparent, because no test is executed against the generated parse since the whole purpose of the test case was to make sure, the grammar is actually successfully converted to a parser.